src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/CwdsDAO.scala

```diff
@@ -9,10 +10,17 @@ trait CwdsDAO {
 
   def getSupportedFormats: List[String]
 
+  @throws(classOf[ApiException])
   def listJobsV1(workspaceId: String,
                  runningOnly: Boolean
                 )(implicit userInfo: UserInfo): List[ImportServiceListResponse]
 
+  @throws(classOf[ApiException])
+  def getJobV1(workspaceId: String,
+               jobId: String
+              )(implicit userInfo: UserInfo): ImportServiceListResponse
+
+  @throws(classOf[ApiException])
```

Comment:
Scala is just fine without the `@throws` annotation, but Mockito needs them to throw checked exceptions out of mocks.